packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -69,108 +75,148 @@ const defaultProps = {
   onNext: null,
   onPrevious: null,
   children: null,
+  locale: 'en-US',
```

Comment:
I'd talk to some of the i18n people, but I thought I remembered someone saying we shouldn't default locale's in one of our meetings